### DISCUSSION TITLE
Contributing to Dogecoin

### DISCUSSION THREAD

**Discussion by rnicoll** [dogecoin/dogecoin#2257] [Help needed]
This discussion board is both for the devs to try directing people towards work that needs attention, and also for people to ask if they get stuck (I may split these later).

Please read the contribution guide as well, which is currently at https://github.com/dogecoin/dogecoin/blob/master/CONTRIBUTING.md

**Comment by Sttarrlord** [comment]
Is there dogecoin miner available. That seems a problem to check and mine on the core. I would like to be part of this process.

**Comment by Doge369420** [comment]
@rnicoll That link needs updating.

**Reply by patricklodder** [reply]
Fixed that for y'all. Thanks!
